SubscriptionStore_test.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"testing"
+	"time"
+
+	_ "github.com/jackc/pgx/v4/stdlib"
+	"github.com/shopspring/decimal"
+)
+
+func TestDatabaseConnection(t *testing.T) {
+
+	t.Run("tests a successful database connection", func(t *testing.T) {
+		_, err := NewSubscriptionStore("user=charlotte  host=localhost port=5432 database=subscryptdb sslmode=disable")
+
+		if err != nil {
```

Comment:
We could look at separating the error checks to separate methods for re-use in the tests